benchmarks/submit_order.js

```diff
@@ -0,0 +1,13 @@
+'use strict'
+
+const { setupBenchmark } = require('./utils/setup')
+const { spawnHost, connectHost } = require('./fixtures/host')
+
+const { performOrder } = require('./fixtures/orders')
+const host = spawnHost()
+const connectPromise = connectHost(host)
+
+setupBenchmark(async () => {
+  await connectPromise
+  await performOrder(host)
```

Comment:
just to confirm, the host will be implicitly closed when the process finishes or we have to add that?

Reply:
The WS server will receive `close` event whenever its client's process has terminated, right?

Reply:
I think we are good, please disregard that